frontend/src/app/components/MainInstallButton.js

```diff
@@ -54,6 +54,7 @@ export default class MainInstallButton extends React.Component {
       installButton = this.renderOneClickInstallButton(experimentTitle);
     } else {
       installButton = <span className="default-btn-msg" data-l10n-id="landingInstallButton">
+        Install the Test Pilot Add-on
```

Comment:
Is this one of the bits where we removed the DOM text because it led to conflicts between React & L20n?